### PR TITLE
LG-4341: Add 72px spacing token

### PR DESCRIPTION
**Pull request by stephl3**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-4341](https://jira.mongodb.org/browse/LG-4341)

### For bug fixes, new features & breaking changes

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

**Review by bruugey — APPROVED** [review]

File: packages/tokens/src/spacing.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ const spacing = {
   1200: 48,
   1400: 56,
   1600: 64,
+  1800: 72,
```
Comment (bruugey) [inline packages/tokens/src/spacing.ts:18]: is this the right number... it's only 6 away from 1600 so i think it shoudl be 1750 not 1800

**Review by bruugey — CHANGES_REQUESTED** [review]
i don't think the number is correct

File: packages/tokens/src/spacing.ts (hunk 1 above)
Comment (TheSonOfThomp) [reply packages/tokens/src/spacing.ts:18]: This should be correct: 18 x 4 = 72

**Review by TheSonOfThomp — APPROVED** [review]

File: .changeset/sweet-rice-unite.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/tokens': minor
+---
+
+Adds 72px spacing token
```
Comment (TheSonOfThomp) [inline .changeset/sweet-rice-unite.md:5]: nit: maybe add what the token key is here

File: packages/tokens/src/spacing.ts (hunk 1 above)
Comment (bruugey) [reply packages/tokens/src/spacing.ts:18]: ugh you're right

Comment (stephl3) [reply packages/tokens/src/spacing.ts:18]: @bruugey it's 8 away

1800 / 100 = 72 / 4 = 18

**Review by bruugey — APPROVED** [review]

**Pull request merged**